static/docs/commands-reference/install.md

```diff
@@ -52,6 +52,21 @@ The installed Git hook automates executing `dvc push`.
 - Git `pre-push` hook executes `dvc push` before `git push` to upload files and
   directories under DVC control to remote.
 
+For more information about git hooks, refer to the
+[git-scm documentation](https://git-scm.com/docs/githooks).
+
+## Disable Git hooks
+When you run `dvc install`, it creates three files under the `.git/hooks` directory:
+
+```
+.git/hooks
+├── post-checkout
+├── pre-commit
+└── pre-push
+```
+
+To disable them, you need to remove those files (i.e. `rm .git/hooks/post-checkout`).
```

Comment:
are we sure that those files are DVC-specific? do we overwrite them when we install the hooks or append to them?

Reply:
We append to them, @shcheklein , I'll add a note about it.